package.json

```diff
@@ -722,27 +722,24 @@
             "label": "%cmake-tools.debugger.label%",
             "configurationAttributes": {
                 "launch": {
-                    "required": [
-                        "debuggerPipeName"
-                    ],
                     "properties": {
-                        "debuggerPipeName": {
+                        "pipeName": {
```

Comment:
Is anyone using this feature yet?  You may want to include temporary support for the old names (with a log message to ask them to migrate) so that they don't get broken unexpectedly.  If this feature is only in pre-release, there's less of an impact, but it could smooth the experience out for early adopters.

Reply:
There are a small number of people, we have gotten some issues posted about it. @benmcmorran and I had talked about this and felt like since it's still in pre-release, we aren't making any gaurantees, so we wanted to do this before we have lots of users and especially before we get it into the official release.